Makefile

```diff
@@ -25,6 +25,7 @@ envinit:
 	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.27.0
 	GO111MODULE=off go get -u github.com/stretchr/testify # needed for mockery
 	GO111MODULE=off go get -u github.com/vektra/mockery/...
+	GO111MODULE=off go get -u github.com/matryer/moq
```

Comment:
What about the mockery just above moq?

mockery:

> mockery provides the ability to easily generate mocks for golang interfaces

moq:

> Interface mocking tool for go generate.

I thought one would be more than enough?

Reply:
mockery makes a different kind of mock struct, specifically to be used with testify mock.

moq creates them like we use to write them by hand: one struct field function per interface method.

i haven't settled yet on which is better, currently i prefer the latter ;)